lib/corm/model.rb

```diff
@@ -138,6 +140,51 @@ def self.count
       execute(statements['count']).first['count'].to_i
     end
 
+    ##
+    # Find by keys.
+    # This `find` methods wants to be as flexible as possible.
+    #
+    # Unless a block is given, it returns an `Enumerator`, otherwise it yields
+    # to the block an instance of the found Cassandra entries.
+    #
+    # If no keys as passed as parameter, the methods returns (an Enumerator for)
+    # all the results in the table.
+    #
+    # If one or more keys are given, the **order matters**; since this methods
+    # will craft a CSQL query, the first key **must** be the `partition key`,
+    # while the followings are intended as the `clustering keys`... that can be
+    # more than one.
+    #
+    # If the keys passed as parameter are more than the defined by the table the
+    # query is not valid, it cannot be executed and an error is raised.
+    def self.find(*key_values, &block)
+      return to_enum(:find, *key_values) unless block_given?
+
+      find_key = "find_#{Digest::MD5.hexdigest(key_values.inspect)}"
```

Comment:
Perfectly clear... TBH that line was quite a bit a "_sudden gut reaction_" after a long day ... I'll fix it!
Thanks!

:sweat_smile: